src/main/kotlin/com/team4099/robot2022/commands/intake/PrepareClimbCommand.kt

```diff
@@ -0,0 +1,26 @@
+package com.team4099.robot2022.commands.intake
+
+import com.team4099.lib.logging.Logger
+import com.team4099.robot2021.subsystems.com.team4099.robot2022.subsystems.Intake
+import com.team4099.robot2022.config.Constants
+import edu.wpi.first.wpilibj2.command.CommandBase
+
+class PrepareClimbCommand : CommandBase() {
```

Comment:
idk for sure but climb may need intake to be extended this year depending on how far the traversal climb thing has to rotate to tilt  the robot far enough